openmdao/docs/other/auto_ivc_api_translation.rst

```diff
@@ -81,5 +84,159 @@ Declaring a Multi-Component Input as a Design Variable
+      MyComp1
+      run_model
+      style2
+      
+        class MyComp1(om.ExplicitComponent):
+            def setup(self):
+                # this input will connect to entries 0, 1, and 2 of its source
+                self.add_input('x', np.ones(3), src_indices=[0, 1, 2])
+                self.add_output('y', 1.0)
+
+            def compute(self, inputs, outputs):
+                outputs['y'] = np.sum(inputs['x'])*2.0
+
+        class MyComp2(om.ExplicitComponent):
+            def setup(self):
+                # this input will connect to entries 3 and 4 of its source
+                self.add_input('x', np.ones(2), src_indices=[3, 4])
+                self.add_output('y', 1.0)
+
+            def compute(self, inputs, outputs):
+                outputs['y'] = np.sum(inputs['x'])*4.0
+
+        p = om.Problem()
+
+        p.model.add_subsystem('indep', om.IndepVarComp('x', np.ones(5)),
+                              promotes_outputs=['x'])
+        p.model.add_subsystem('C1', MyComp1(), promotes_inputs=['x'])
+        p.model.add_subsystem('C2', MyComp2(), promotes_inputs=['x'])
+
+        p.model.add_design_var('x')
+        p.setup()
+        p.run_model()
+        
+
+Setting Default Units for an Input 
+==================================
+
+.. content-container ::
+
+  .. embed-compare::
+      openmdao.test_suite.tests.test_auto_ivc_api_conversion.TestConversionGuideDoc.test_units
+      Problem
+      setup
+      style2
+      
+        prob = om.Problem()
+        
+        ivc = om.IndepVarComp()
+        ivc.add_output('x2', 100.0, units='degC')
+        prob.model.add_subsystem('T1', ivc,
+                                 promotes_outputs=['x2'])
+        
+        # Input units in degF
+        prob.model.add_subsystem('tgtF', TgtCompF(), 
+                                 promotes_inputs=['x2'])
+
+        # Input units in degC
+        prob.model.add_subsystem('tgtC', TgtCompC(), 
+                                 promotes_inputs=['x2'])
+
+        # Input units in deg
+        prob.model.add_subsystem('tgtK', TgtCompK(), 
+                                 promotes_inputs=['x2'])        
+        
+        prob.setup()
+        
+        
+        
+Creating a Distributed Component with Unconnected Inputs
+========================================================
+
+.. content-container ::
+
+  .. embed-compare::
+      openmdao.test_suite.tests.test_auto_ivc_api_conversion.TestConversionGuideDocMPI.test_prob_getval_dist_par
+      size
+      run_model
+
+        size = 3
+
+        p = om.Problem()
+        top = p.model
+        par = top.add_subsystem('par', om.ParallelGroup())
+        C1 = par.add_subsystem("C1", DistribInputDistribOutputComp(arr_size=size))
+        C2 = par.add_subsystem("C2", DistribInputDistribOutputComp(arr_size=size))
+        p.setup()
+
+        # Conclude setup but don't run model.
+        p.final_setup()
+
+        if C1 in p.model.par._subsystems_myproc:
```

Comment:
can you not use the API function `system_iter` here instead of the underscored attribute?

Reply:
Working on a new, simpler example to replace this one.

Reply:
Done